docs/dev/table/connectors/jdbc.md

```diff
@@ -183,7 +183,14 @@ Connector Options
       <td style="word-wrap: break-word;">0</td>
       <td>Integer</td>
       <td>The number of rows that should be fetched from the database when reading per round trip. If the value specified is zero, then the hint is ignored.</td>
-    </tr>     
+    </tr>
+    <tr>
+      <td><h5>scan.auto-commit</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>Boolean</td>
+      <td>Sets the auto commit flag on the JDBC driver.</td>
```

Comment:
Maybe it would be better to have a link to explain what is the auto commit? For example: https://docs.oracle.com/javase/tutorial/jdbc/basics/transactions.html#commit_transactions? 

However the above link only explains the behavior for DMLs, maybe some explanation for what it is used for scan would be better.

Reply:
Good idea. I had actually at first made a comment about the postgres JDBC driver behavior with scans, since I have run across this multiple times, but had removed it since it was vendor-specific. Any thoughts about including a comment about that?

Reply:
I think the purpose of this option is addressing some specific database requirement. So having postgres as an example is great for users to understand.